meta/main.yml

```diff
@@ -11,7 +11,6 @@ galaxy_info:
     - focal
   - name: Debian
     versions:
-    - stretch
```

Comment:
If `stretch` is removed here then `... Debian 9 (Stretch), ...` should also be removed from README.

Reply:
Forgot about that, thanks. I fixed up the commit that to keep atomic commits.

I also rebased on latest master to resolve the conflicts.